release-1.9/release-1.9.md

```diff
@@ -0,0 +1,100 @@
+- Oct 25th (Weds): v1.9.0-alpha.2
+- Oct 27th (Fri): Feature Freeze
+- Nov 8th (Weds): v1.9.0-alpha.3
+- Nov 22nd (Weds): v1.9.0-alpha.4
+- Nov 22nd (Weds): Code Freeze 
+- Nov 29th (Weds): v1.9.0-beta.0, v1.10.0-alpha.0
+- Dec 6th (Weds): v1.9.0-beta.release-candidate
+- Dec 8th (Mon): v1.10.0-alpha.1
+- Dec 13th (Weds):  v1.9.0 release!
+
+## 1.9 Details
+
+### Oct 5 - Nov 22
+- 8 week coding period
+- Release 1.9 alphas every 2 weeks
+  - Alphas only cut if [release-master-blocking](https://k8s-testgrid.appspot.com/release-master-blocking) are passing
+- Oct 11th (Weds): v1.9.0-alpha.1
+- Oct 25th (Weds): v1.9.0-alpha.2
+- Oct 27th (Fri): 
+    * Feature Freeze: all features going into release should have an associated issue in the features repo
+    * Release talking points draft due
+    * Sig themes due in relnotes draft
+- Oct 31st (Tues): 
+    * First draft of feature related release notes due
+    * Blog post draft for release team review
+- Nov 8th (Weds): v1.9.0-alpha.3
+- Nov 15th (Weds): 
+    * Final draft of feature related release notes due
+    * Finalize release talking points
+- Nov 22nd (Weds): v1.9.0-alpha.4
+
+
+### Nov 22 (Wed) - Dec 8 (Weds)
+- Nov 22 Begin Code Slush
+  * Hard deadline for feature-related PRs to be in submit queue with LGTM and approved label
+  * Add milestone restriction on submit queue.
+  * Community Feature Burndown Meetings held two or three times until release week (then every day). For those interested in joining please join [Kubernetes Milestone Burndown Group](https://groups.google.com/forum/#!forum/kubernetes-milestone-burndown)
+  * Focus on bugfix, test flakes and stabilization
+  * Ensure docs and release notes are written
+  * Identify all features going into the release, and make sure alpha, beta, ga is marked in features repo
+  * Release team to create release notes draft doc and share broadly
+- Nov 29th (Weds): v1.9.0-beta.0
+  * Branch Manager to create `release-1.9` branch.
+  * v1.10.0-alpha.0 is cut
+  * Test-infra lead to set up CI for release branch.
+  * Begin regular fast-forwards (at least daily to keep CI up-to-date)
+- Dec 1 (Fri)
+  * All docs PRs must be open for tech review
+
+### Dec 8th - Dec 13th
+- Dec 8th (Weds): End of Code Freeze
+  * v1.9.0-beta.release-candidate cut
+	  * RC means no known release-blockers outstanding.
+	  * Only accept cherrypicks for release-blockers.
+	  * Announce on mailing lists, Twitter, etc. to ask people to build and test the RC and submit feedback
+	  * Perhaps managed k8s providers make rc.1 available on early access channels.
+  * More RCs as needed to verify release-blocking fixes.
+  * Final fast-forward of release branch.
+  * After this, all changes for the release must now be cherrypicked in batches by branch
+  manager.
+  * Remove milestone restriction on submit queue. 
+  * Bot drains backlog.
+  * Finalize blog post
+- Dec 8th (Fri)
+  * Docs PRs lgtm'd and ready for merge
+- Dec 13th: Release 1.9!
+
+
+### Post Release
```

Comment:
Should we talk about when blog posts / other marketing material etc. are published as well?

Reply:
+1.